lc0_main.go

```diff
@@ -213,8 +214,9 @@ func (c *cmdWrapper) launch(networkPath string, args []string, input bool) {
 			for _, word := range words {
 				optOK := false
 				switch word {
-					case "", "--backend", "tf", "cudnn", "opencl", "blas",
-						"--backend-opts", "gpu", "verbose", "true", "false":
+					case "", "--backend", "tf", "cudnn", "opencl", "blas", "cudnn-fp",
```

Comment:
Have you tested this?  I got an error from the backend that it couldn't parse a number or something when I added cudnn-fp

Reply:
the rest of the change is fine, we can sort out the number parsing later.